quiz.js

```diff
@@ -31,3 +31,38 @@ function tree(treeLimbs) {
 
 
 
+=======
+field.addEventListener("keyup", function(keyPress){ // Event listener on text field for 'enter' keypress
+	var chars = document.getElementById("field").value
+	keyPress.preventDefault();
+	if (chars.length > 1) {
+		alert("Enter just one character, plz."); // Prevents user from entering more than one character.
+	} else if (keyPress.keyCode == 13) {
+		document.getElementById("grow").click();
+	};
+});
+
+height.addEventListener("keyup", function(keyPress){ // Event listener on text field for 'enter' keypress
+	var integer = document.getElementById("height");
+	keyPress.preventDefault();
+	if(keyPress.keyCode < 48 && keyPress.keyCode !== 8 || keyPress.keyCode > 57 && keyPress.keyCode != 8) {
+		alert("Enter only integers, plz."); // Prevents user from entering anything other than an integer
+	} else if (keyPress.keyCode == 13) {
+		document.getElementById("grow").click();
+	};
+});
+
+function tree(treeLimbs) {
+	var treeLimbs = { // object to hold user input
```

Comment:
This object should be built (ie have properties and values) before it's passed into the function.